source/plugin/events.rst

```diff
@@ -252,7 +284,7 @@ Example: Adding a Callback to Disable Explosions and Spawn an Arrow
 
 .. code-block:: java
 
-    @Subscribe
+    @Listener
     // final not required unless using an inner class that needs access to it.
     public void onExplosion(final ExplosionEvent event) {
```

Comment:
Usually the format is onSomethingEvent or onSomethingEventAsPlayer (if one cause is gonna be checked)

onBreakBlockEvent

onBreakBlockEventAsPlayer